guide/additional-info/changes-in-v13.md

```diff
@@ -0,0 +1,1280 @@
+
+`channel.messages.pin('765432109876543210', options)`
+
+Approximately equivalent to `message.pin(options)` but does not resolve to a Message.
+
+#### MessageManager#react
+
+`channel.messages.react('765432109876543210', emoji)`
+
+Approximately equivalent to `message.react(emoji)` but does not resolve to a MessageReaction.
+
+#### MessageManager#unpin
+
+`channel.messages.unpin('765432109876543210', options)`
+
+Approximately equivalent to `message.unpin(options)` but does not resolve to a Message.
+
+### MessageMentions
+
+#### MessageMentions#repliedUser
+
+Checks if the author of a message being replied to has been mentioned.
+
+### MessagePayload
+
+This class has been renamed from APIMessage.
+
+Global headers can now be set in the HTTP options.
+
+### MessageSelectMenu
+
+A builder class which makes constructing select menu type Message Components easy.
+
+### NewsChannel
+
+#### NewsChannel#addFollower
+
+`channel.addFollower(channel)`
+
+Provides API support for bots to follow announcements in other channels.
+
+#### NewsChannel#setType
+
+`channel.setType('text')`
+
+Allows conversion between NewsChannel and TextChannel.
+
+### Permissions
+
+#### Permissions#STAGE_MODERATOR
+
+Static bitfield representing the permissions required to moderate a Stage Channel.
+
+### PermissionOverwriteManager
+
+Replaces the `createOverwrite`, `updateOverwrite` and `overwritePermissions` methods of `GuildChannel`, aligning the design with other Managers.
+
+### Role
+
+#### Role#tags
+
+Tags for roles belonging to bots, integrations, or premium subscribers.
+
+### RoleManager
+
+#### RoleManager#botRoleFor
+
+`guild.roles.botRoleFor(user)`
+
+Gets the managed role a bot created when joining the guild, if any.
+
+#### RoleManager#edit
+
+`guild.roles.edit('12345678987654321', options)`
+
+Equivalent to `role.edit(options)`.
+
+#### RoleManager#premiumSubscriberRole
+
+`guild.roles.premiumSubscriberRole`
+
+Gets the premium subscriber (booster) role for the Guild, if any.
+
+### SelectMenuInteraction
+
+Provides gateway support for a `MessageComponentInteraction` coming from a select menu component.
+
+### StageChannel
+
+Provides API support for Stage Channels.
+
+### StageInstance
+
+Provides API support for Stage Instances. Stage instances contain information about live stages.
+
+### StageInstanceManager
+
+Provides API support for the bot to create, edit and delete live Stage Instances, and stores a cache of Stage Instances.
+
+### Sticker
+
+Provides API support for Discord Stickers.
+
+### StickerPack
+
+Provides API support for Discord Sticker packs.
+
+### TextChannel
+
+#### TextChannel#awaitMessageComponent
+
+A shortcut method to create a promisified `InteractionCollector` which resolves to a single `MessageComponentInteraction`.
+
+#### TextChannel#createMessageComponentCollector
+
+A shortcut method to create an `InteractionCollector` for components on a specific channel.
+
+#### TextChannel#setType
+
+`channel.setType('news')`
```

Comment:
Is this still 'news' or has it been changed to
```suggestion
`channel.setType('GUILD_NEWS')`
```
?

Reply:
`GUILD_NEWS` is correct